pkg/envfuncs/kind_funcs.go

```diff
@@ -205,3 +205,26 @@ func LoadImageArchiveToCluster(name, imageArchive string) env.Func {
 		return ctx, nil
 	}
 }
+
+// ExportKindClusterLogs returns an EnvFunc that
+// retrieves a previously saved kind Cluster in the context (using the name), and then export cluster logs
+// in the provided destination.
+func ExportKindClusterLogs(name, dest string) env.Func {
```

Comment:
If you could update the usage of this function under examples/kind that would be great.

Reply:
Hi! Thanks! Example is now updated !